cloudflare/src/handle.rs

```diff
@@ -24,70 +23,64 @@ lazy_static! {
 ///
 pub async fn fetch(req: worker::Request, env: worker::Env, _: worker::Context) -> anyhow::Result<worker::Response> {
   log::info!("{} {:?}", req.method().to_string(), req.url().map(|u| u.to_string()));
-  let env = Rc::new(env);
-  let hyper_req = to_request(req).await?;
-  if hyper_req.method() == hyper::Method::GET {
-    let response = graphiql(&hyper_req)?;
+  let req = to_request(req).await?;
+  if req.method() == hyper::Method::GET {
```

Comment:
I don't think we need to handle this here. The behavior should be controlled by the core module.

Reply:
Had to re-add this. When GraphiQL is requested, a config parameter is not specified, and `get_app_ctx` fails without it, so we don't actually get to the point where `handle_request` could route to `graphiql`.